netbox_attachments/template_content.py

```
@@ -1,74 +1,70 @@
 
 logger = logging.getLogger(__name__)
-PLUGIN_SETTINGS = settings.PLUGINS_CONFIG.get("netbox_attachments", {})
 
 
-def render_attachment_panel(self) -> str:
-    """
-    Renders the attachment panel template for a model.
+def _get_plugin_settings():
+    try:
+        plugins_config = getattr(settings, "PLUGINS_CONFIG", {})
+    except Exception:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)